.clang-format

```diff
@@ -0,0 +1,72 @@
+AccessModifierOffset: -4
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: true
+AlignConsecutiveDeclarations: true
+AlignEscapedNewlinesLeft: Right
```

Comment:
I don't get that does this option do :(

Reply:
Something like this.
```
#define A   \
  int aaaa; \
  int b;    \
  int dddddddddd;
```

Reply:
What if we have something like:
```cpp
bool a;
const uint8_t * b;
```
For this case alignment would be weird :)

Reply:
It's only about the escape `\` symbol like in macros or comments.

Reply:
Then it's totally fine. We use alignment for macros like this.